### DISCUSSION TITLE
Octernship Unable to apply. Help please

### DISCUSSION THREAD

**Discussion by hisandan** [community/community#51625] [GitHub Education]
I am trying to click on the I'm interested button of an Octernship, however after that it shows me
 `Maximum application limit reached! You cannot apply to this octernship at this moment.`

 and just below it it says
 `9 projects available. You can still apply to 4`

 Could you give me some advice on what may be happening please?

**Comment by Ntshangase** [comment]
Hey @danisshai 

Have you applied to any projects before?

**Comment by hisandan** [comment]
Hi @Ntshangase , no 🙁, this was my first.

**Reply by Ntshangase** [reply]
Then it means the project you are interested in is currently full, but you can apply for the other 9 available projects.

**Reply by samus-aran** [reply]
Awesome reply @Ntshangase thanks for sharing the knowledge and helping out 👍

**Reply by hisandan** [reply]
Thank you

**Reply by hisandan** [reply]
> en it means the project you are interested in is currently full, but you can apply for the other 9 available projects.
